src/main/kotlin/com/mjucow/eatda/presentation/store/StoreApiPresentation.kt

```diff
@@ -0,0 +1,43 @@
+package com.mjucow.eatda.presentation.store
+
+import com.mjucow.eatda.common.dto.CursorPage
+import com.mjucow.eatda.domain.store.service.command.dto.MenuCreateCommand
+import com.mjucow.eatda.domain.store.service.command.dto.StoreCreateCommand
+import com.mjucow.eatda.domain.store.service.command.dto.StoreUpdateCommand
+import com.mjucow.eatda.domain.store.service.query.dto.MenuList
+import com.mjucow.eatda.domain.store.service.query.dto.StoreDetailDto
+import com.mjucow.eatda.domain.store.service.query.dto.StoreDto
+import com.mjucow.eatda.presentation.common.ApiResponse
+import com.mjucow.eatda.presentation.common.example.StoreDtosApiResponse
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+
+@Tag(name = "가게 API", description = "가게를 관리해주는 API")
+interface StoreApiPresentation {
+
+    @Operation(summary = "가게 생성", description = "가게를 생성합니다.")
+    fun create(command: StoreCreateCommand): ApiResponse<Long>
+
+    @Operation(summary = "커서 기반 카테고리 가게 조회", description = "커서를 기반으로 가게를 조회합니다.")
+    @StoreDtosApiResponse
+    fun findAllByCategoryIdAndCursor(
+        storeId: Long?,
+        categoryId: Long?,
+        pageSize: Int?,
```

Comment:
여기 default value가 안 들어가는거죠?

Reply:
이런 식으로 기본 값 할당은 안되나요?
```suggestion
        pageSize: Int = 20,
```